src/k8s-engine/lima.ts

```diff
@@ -147,12 +155,19 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
   /** The port the Kubernetes server _should_ listen on */
   #desiredPort = 6443;
 
+  /** Currently either containerd or moby, changing requires a full restart */
```

Comment:
```suggestion
  /** The current container engine; changing this requires a full restart. */
```
(I blame Thulio for making me always put a full stop at the end of comments.)